app/src/routes/api/v1/subscription.router.js

```diff
@@ -300,18 +314,48 @@ const subscriptionExists = (isForUser) => async (ctx, next) => {
     await next();
 };
 
-const isLoggedUserRequired = async (ctx, next) => {
+const checkMicroservice = (ctx) => {
```

Comment:
if the result is a boolean, name the function as such, so it's easier to understand what it does. A good rule of thumb is: if your function return a boolean, its name should start with "is" or "has"

Reply:
Agree, though I should add (at least to me) I include "check" as a valid name for methods that return booleans. But, to keep consistency with what we have already in place, changed it to `isMicroservice` :+1: